visitors/http_rest/field_location.go

```diff
@@ -0,0 +1,75 @@
+package http_rest
```

Comment:
Can this go in `field_utils/field_location.go` (and update it to use the new path elements)?  It would be nice to have a single notion of FieldLocation.

https://github.com/akitasoftware/superstar/blob/c40faa6300b40ff0ea03721da290f20931d324d8/lib/field_utils/field_location.go

Reply:
Or call this something like `type RelativeFieldLocation FieldLocationElement[]` and update the other `FieldLocation` struct to replace `path` with`relativeLocation: RelativeFieldLocation`?

Reply:
Ideally, we'd unify the two, but is this really the data structure we want to export to the front end?

Reply:
To be clear, the `FieldLocationElement`s would be replacing the strings in this array: https://github.com/akitasoftware/superstar/blob/c40faa6300b40ff0ea03721da290f20931d324d8/lib/field_utils/field_location.go#L35-L40

Reply:
Yea, fair enough -- ideally we'd unify them and write a nice serializer, but it's not worth the effort right now.